source/Calamari.AzureWebApp/Calamari.AzureWebApp.csproj

```diff
@@ -5,8 +5,8 @@
         <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
         <IsPackable>false</IsPackable>
         <OutputType>Exe</OutputType>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
         <RuntimeIdentifiers>win-x64;linux-x64;osx-x64;linux-arm;linux-arm64</RuntimeIdentifiers>
+        <TargetFrameworks>net452;netcoreapp3.1</TargetFrameworks>
```

Comment:
`Calamari.Azure.WebApps` was [previously](https://github.com/OctopusDeploy/Calamari/blob/2fa68396795697f6419a599f7c87888342bf8142/source/Calamari.Azure.WebApps/Calamari.Azure.WebApps.csproj#L5) only built for `net452`, not net core. My understanding was that this is due to Linux not having the required libs (e.g. web deploy).